### DISCUSSION TITLE
Glow_x64.exe with parameters via the command line interface or powershell

### DISCUSSION THREAD

**Discussion by ChrisNowa2020** [turkaysoft/glow#24] [Ideas]
Dear Mr. Eray Türkay,

Thanks for your software solution glow, I came across your software by chance on github.

I would like to politely ask whether it would be possible in the future to start Glow_x64.exe with parameters via the command line interface or powershell

example: 
Glow_x64.exe -de -report " R:\Glow_%Computername%_Report.html
Glow_x64.exe -en -report " R:\Glow_%Computername%_Report.html

Greetings Chris

**Comment by turkaysoft** [comment]
Hello,

It's hard to say for sure. It will be a nice feature, of course, but it would not be right to give an exact date. I would like to add it when possible, of course.

**Reply by turkaysoft** [reply]
Hello @ChrisNowa2020 

We have laid the groundwork to implement this proposal.
We are currently researching what we can do for the development process.

Glow.exe will support both the user interface and console commands.

**Reply by turkaysoft** [reply]
Hello, unfortunately this feature will not be implemented.
At the moment, the Glow modern interface infrastructure is appreciated by users.
